docs/src/utils/getStage.ts

```diff
@@ -0,0 +1,15 @@
+enum STAGE {
+  LOCAL = 'local',
+  DEV = 'dev',
+  PROD = 'prod',
+}
+
+export const getStage = () => {
+  const url = process.env.SITE_URL;
+  if (url === 'https://ui.docs.amplify.aws') {
+    return STAGE.PROD;
+  } else if (url.includes('dev.ui.docs.amplify.aws')) {
+    return STAGE.DEV;
+  }
+  return STAGE.LOCAL;
+};
```

Comment:
Can we export IS_PROD_STAGE and use that instead?
```ts
const IS_PROD_STAGE = getStage() === 'prod'
```

Reply:
Done